planetarium/NineChronicles
# Feature/improve notification and mail message of combination or enhancement

### Description

1. SSIA

### How to test

1. Try combine or enhance items.
2. Check notify and mail message when step#1 finished.

### Related Links

1. [[제작&강화] 결과 알림 적용하기](https://app.asana.com/0/958521740385861/1200727868859771/f)
